python/example_robot_data/robots_loader.py

```diff
@@ -239,6 +239,50 @@ def loadTalosArm():
     return loadTalos(arm=True)
 
 
+class CassieLoader(RobotLoader):
+    path = 'cassie_description'
+    sdf_subpath = 'robots'
+    sdf_filename = "cassie_v2.sdf"
+    free_flyer = True
+    verbose=False
+
+    def __init__(self):
+        sdf_path = join(self.path, self.sdf_subpath, self.sdf_filename)
+        self.model_path = getModelPath(sdf_path, self.verbose)
+        self.sdf_path = join(self.model_path, sdf_path)
+        self.robot = RobotWrapper.BuildFromSDF(self.sdf_path, [join(self.model_path, '../..')],
+                                                pin.JointModelFreeFlyer() if self.free_flyer else None)
+        q0 = pin.neutral(self.robot.model)
+        q0[2] = 1.104;    q0[9] = 0.298;    q0[20] = 0.298;    q0[17] = -1.398
```

Comment:
I thought you already modified the robot. Why not updating it with the data you need ?
I think robot description data belong to urdf / srdf / sdf files, while python files should only be there to reduce boilerplate

Also, this is breaking the format CI with:
`./python/example_robot_data/robots_loader.py:256:22: E702 multiple statements on one line (semicolon)`

Reply:
I modified the kinematic tree in the original robot sdf, but not it's initial configuration. I agree initial configuration belongs to `srdf`, but I don't think I would have enough time to create an srdf with cassie. If you think there is a stop-gap measure, we can use that, or keep this PR open till I can push an `srdf`